README.md

```diff
@@ -2,24 +2,251 @@
+* Nominate and approve the appointment of individuals to fill the role of Release Manager (per release), as described below.
 
-If you have a question about the PMIx interface or want to raise a question for discussion then please file an Issue against this repository. We are using the Issues to discuss topics, and will link Pull Requests related to that topic to that Issue. This provides the community with a public, archived forum for discussion.
+The ASC is led by two Co-Chairs responsible for organizing and conducting the quarterly meetings. This includes:
 
-### Participatin in the writing
+* Announcing meeting agendas at least one month in advance of the meeting. Any proposal wishing to be considered for the next meeting must be put forward (by assigning the proper label, as described below) prior to that time. The ASC Co-Chairs may allow a proposal to be added to the agenda after this time on a case-by-case basis at their sole discretion.
+* Chairing the quarterly meetings and ensuring an opportunity for all to express their opinion during the meeting
+* Maintaining and publishing minutes of the meeting, and tallying votes.
 
-If you want to write a portion of the standard (or even fix a typo or two), then use the Pull Request mechanism to post the change for review.
+Co-Chairs may vote on behalf of their organization subject to the one vote per Member organization limit. Co-Chairs serve for a period of two years, with seats filled on alternate years - i.e., one seat will be elected during the first meeting of each even year, and the other seat will be elected during the first meeting of each odd year. There is no restriction on the number of consecutive terms a given individual may serve - however, it is expected and desirable that the positions rotate amongst the Members.
 
-If you want plan to work on a large section of the document then please coordinate with the community on the PMIx mailing list to make sure we are not duplicating efforts. When working on a large section (particularly if the changes are going to take a day or two to make) then let the community know, file a PR with your first set of changes, and mark the PR as "Work In Progress".  Once it is ready for review then remove the "Work In Progress" label, and let the community know that it's ready for discussion.
+Participation in the consortium is both formal and informal:
 
-## Building the PMIx Standard document
+* Informal participation is encouraged by working with the PMIx Standard community and/or one of the PMIx implementation libraries in an _ad hoc_ manner, such as (but not limited to) engaging in discussions on the mailing lists, reporting testing results, providing feature requests and bug reports, filing pull requests for proposed modifications, etc. Informal participants do not have voting privileges on administrative matters, but are encouraged to attend the quarterly meetings and participate in those discussions.
 
```

Comment:
Should make a note that informal participants can attend meetings and, in fact, must attend meetings to become formal members.   Maybe something such as: "Informal participants may attend quarterly meetings and an individual or organization must attend meetings as informal participants before being elevated to Members."

Reply:
We state that in the following paragraph, but I have no issue with restating it here as well:

> Informal participants do not have voting privileges on administrative matters, but are encouraged to attend the quarterly meetings and participate in those discussions. Individuals and/or organizations must attend meetings as informal participants in order to qualify for elevation to full Member status, as described below.